PokeAlarm/Utils.py

```diff
@@ -148,6 +148,69 @@ def get_move_energy(move_id):
             get_move_energy.info[int(id_)] = j[id_]['energy']
     return get_move_energy.info.get(move_id, 'unkn')
 
+# Returns the base stats for a pokemon
+def get_base_stats(pokemon_id):
```

Comment:
Perhaps rather than have 3 functions, can get_base_height and get_base_width memoize themselves? I think this might make it more clear, and I know this means reading the file twice, but since it only needs to do it once I I don't think performance is an issue.